src/librustc/hir/map/mod.rs

```diff
@@ -738,6 +738,7 @@ impl<'hir> Map<'hir> {
                 | Node::Item(_)
                 | Node::ForeignItem(_)
                 | Node::TraitItem(_)
+                | Node::AnonConst(_)
```

Comment:
You probably shouldn't use `get_parent_item` for this.

Reply:
Agree, but didn't want to duplicate the `get_parent_item` code just yet before I could test this would be enough.